controllers/drplacementcontrol_controller.go

```diff
@@ -1480,13 +1590,43 @@ func (d *DRPCInstance) getVRGFromManifestWork(mw *ocmworkv1.ManifestWork) (*rmn.
 func (d *DRPCInstance) updateDRPCStatus() error {
 	d.log.Info("Updating DRPC status")
 
-	d.instance.Status.LastUpdateTime = metav1.Now()
+	drpc := &rmn.DRPlacementControl{}
+	drpcLookupKey := types.NamespacedName{
+		Name:      d.instance.Name,
+		Namespace: d.instance.Namespace,
+	}
+
+	err := d.reconciler.Client.Get(d.ctx, drpcLookupKey, drpc)
```

Comment:
We should not be getting the latest instance and updating the status, is there an issue here?

Reply:
No, we shouldn't.  That's overkill.  I added that because the unit test was modifying the object, and it ended up failing here.  I'll remove it and see how to fix it somewhere else.